app/scripts/museum/museum-service.js

```diff
@@ -100,6 +106,30 @@
             return Util.makeRequest(sql, query);
         }
 
+        function listByCity(params) {
+            var attributes = { 'gcity': 'city', 'gstate': 'state', 'gzip': 'zip' };
+            var whereArray = _(attributes)
+                .mapValues(function (v) { return params[v]; })
+                .pick(function (v) { return !!(v); })
+                .map(function (v, k) {
+                    if (k === 'gstate' && v.length !== 2) {
+                        v = getStateAbbrev(v);
+                    }
+                    return Util.strFormat('{k} ILIKE \'%{v}%\'', {k: k, v: v});
```

Comment:
Almost certainly. I _think_ the cartodb sql library handles this case, but I'll double check.